tardis/model.py

```diff
@@ -117,17 +118,23 @@ def __init__(self, tardis_config):
             v_inner=tardis_config.structure.v_inner,
             v_outer=tardis_config.structure.v_outer)
 
-        distance = tardis_config.supernova.get('distance', None)
-        self.spectrum = TARDISSpectrum(
-            tardis_config.spectrum.frequency, distance)
-        self.spectrum_virtual = TARDISSpectrum(
-            tardis_config.spectrum.frequency, distance)
-        self.spectrum_reabsorbed = TARDISSpectrum(
-            tardis_config.spectrum.frequency, distance)
-
         self.calculate_j_blues(init_detailed_j_blues=True)
         self.update_plasmas(initialize_nlte=True)
 
+    @property
+    @deprecated('v1.5', obj_type='attribute')
```

Comment:
can we add a message saying ' this will disappear the spectrum is now at model.runner.xxx'